src/css/lib/monster.ui.scss

```diff
@@ -992,3 +995,42 @@ i.monster-password-strength {
 		font-size: 0.75rem;
 	}
 }
+
+/************************************************************/
+/********************* JQUERY UI WIDGETS ********************/
+/************************************************************/
+.ui-widget.ui-spinner {
```

Comment:
jQuery UI widgets overrides should be in `monster-ui\src\css\lib\custom\jquery-ui-add.scss`